assets/css/admin-notices.scss

```diff
@@ -64,19 +64,19 @@ $sensei-notice-icon-width: 30px;
 		position: static;
 	}
 
-	&.sensei-notice-error {
+	&__error {
```

Comment:
These classes should use `--` to follow the correct standard because they're a modifier.

Reply:
TIL! Thanks for letting me know that, I had no idea. https://github.com/Automattic/sensei/pull/6816/commits/3ad66d68a1213b9b408b87128baf6b13a6de0482